vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/create/ProjectCreationDialog.java

```diff
@@ -151,31 +145,31 @@ private void initForm() {
     formLayout.add(defineExperimentComponent);
     formLayout.add(projectContactsLayout);
     formLayout.add(principalInvestigator);
-    formLayout.add(responsiblePerson);
     formLayout.add(projectManager);
+    formLayout.add(responsiblePerson);
     // set form layout to only have one column (for any width)
     formLayout.setResponsiveSteps(new ResponsiveStep("0", 1));
   }
 
   private void styleForm() {
-    formLayout.setClassName("create-project-form");
+    formLayout.setClassName("project-form-layout");
     styleSearchBox();
-    formLayout.setMaxWidth(60, Unit.VW);
   }
 
   private void styleSearchBox() {
-    offerSearchField.setMaxWidth(30, Unit.VW);
+    offerSearchField.setClassName("search");
```

Comment:
MAJOR NIT: Since the css adaption is tied to the component in this case it makes sense to be more specific here
```suggestion
    offerSearchField.setClassName("search-field");
```